index.js

```diff
@@ -87,10 +87,10 @@ var dic = function (config) {
 
   function mapMirror(key) {
     var map = {
-      "austrailia": "http://www.csse.monash.edu.au/~jwb/cgi-bin/wwwjdic?",
```

Comment:
You update the map but you don't update the row above with the correct keys. On my repo I pulled these out and made this map a const and replaced the row above with map.keys() to keep them forever in sync